ckanext/dcat/plugins.py

```diff
@@ -147,3 +141,15 @@ def get_auth_functions(self):
         return {
             'dcat_datasets_list': dcat_auth,
         }
+
+
+class DCATFacetsPlugin(p.SingletonPlugin, DefaultTranslation):
```

Comment:
As [requested by Adrian](https://github.com/ckan/ckanext-dcat/pull/100#discussion_r147440145), this faceting should not belong here. Please move it to the IT profile.